OpenOversight/app/main/model_view.py

```diff
@@ -35,9 +38,10 @@ def get(self, obj_id):
                     objects = self.model.query.order_by(
                         getattr(self.model, self.order_by)
                     ).paginate(page=page, per_page=self.per_page, error_out=False)
-                objects = self.model.query.order_by(
-                    getattr(self.model, self.order_by).desc()
-                ).paginate(page=page, per_page=self.per_page, error_out=False)
+                else:
+                    objects = self.model.query.order_by(
+                        getattr(self.model, self.order_by).desc()
+                    ).paginate(page=page, per_page=self.per_page, error_out=False)
```

Comment:
This needed to be an in `else` statement, otherwise it overwrote the statement above every time.